docs/changelog.rst

```diff
@@ -1,5 +1,53 @@
 Changelog
 ================================
+CBC SDK 1.5.0 - Released (TBD)
+--------------------------------------
+
+**Breaking Changes:**
```

Comment:
Based on the content below do we want make this section the summary and more explicitly call out the breaking changes